### PR TITLE
Add second instance.

### PR DESCRIPTION
**Pull request by FireMasterK**
And its onion counterpart.

### CODE REVIEW

**Comment by FireMasterK** [comment]
I basically followed https://github.com/iv-org/documentation/commit/cb60363acf3b76c5257bc9fbebed9552762e4a6c which adds it to the same line, if it does break the list, I would be happy to move it to the end.

**Comment by unixfox** [comment]
Mmh ok fair enough. @TheFrenchGhosty maybe you know more the documentation than me, so could you review this PR?

**Comment by TheFrenchGhosty** [comment]
"Secondary" instances are put after the main one, yes it makes them not indexed by https://api.invidious.io/ but it's "okay" (it's an api.invidious.io issue).

I didn't merge it directly because I wanted to associate the onion to its clear-net domain

File: Invidious-Instances.md [hunk 1]
```diff
@@ -58,3 +58,5 @@ To be in this list, instances must have been updated in the last month. An insta
 * [kbjggqkzv65ivcqj6bumvp337z6264huv5kpkwuv6gu5yjiskvan7fad.onion](http://kbjggqkzv65ivcqj6bumvp337z6264huv5kpkwuv6gu5yjiskvan7fad.onion/) 🇳🇱
 
 * [grwp24hodrefzvjjuccrkw3mjq4tzhaaq32amf33dzpmuxe7ilepcmad.onion](http://grwp24hodrefzvjjuccrkw3mjq4tzhaaq32amf33dzpmuxe7ilepcmad.onion) 🇺🇸
+
+* [hpniueoejy4opn7bc4ftgazyqjoeqwlvh2uiku2xqku6zpoa4bf5ruid.onion](http://hpniueoejy4opn7bc4ftgazyqjoeqwlvh2uiku2xqku6zpoa4bf5ruid.onion)
```
Comment (TheFrenchGhosty) [inline Invidious-Instances.md:62]: * [hpniueoejy4opn7bc4ftgazyqjoeqwlvh2uiku2xqku6zpoa4bf5ruid.onion](http://hpniueoejy4opn7bc4ftgazyqjoeqwlvh2uiku2xqku6zpoa4bf5ruid.onion) 🇺🇸 (Onion of invidious-us.kavin.rocks)

Comment (FireMasterK) [reply Invidious-Instances.md]: Done. Albeit, I don't quite understand why this is necessary.

Comment (TheFrenchGhosty) [reply Invidious-Instances.md]: @FireMasterK It's easier for us to track down who own the onions. I plan to refactor the whole instance list soon with new "requirements", this is one of them.

Comment (TheFrenchGhosty) [reply Invidious-Instances.md]: @FireMasterK FYI, this new requirement has been added to the rewrite of the instance list: https://github.com/iv-org/documentation/pull/74

**Pull request merged**